automation/taskcluster/decision_task.py

```diff
@@ -72,10 +72,10 @@ def pr_or_push(is_push=False):
             variant
         )
         if (
-            is_push and
+            # is_push and
```

Comment:
Do you still need these commented-out?

Reply:
That line is gonna be removed. It's part of my temporary commit which I intend to remove.